Assets/Scripts/EOSPlayerDataStorageManager.cs

```diff
@@ -77,27 +55,14 @@ public void QueryFileList()
                 LocalUserId = localUserId
             };
 
-            EOSManager.Instance.GetPlayerDataStorageInterface().QueryFileList(ref options, null, OnFileListRetrieved);
-        }
-
-        //-------------------------------------------------------------------------
-        /// <summary>Add listener callback that will be called when the queried file list is updated.</summary>
-        /// <param name="downloadCompletedCallback">Function called when file list is updated.</param>
-        public void AddNotifyFileListUpdated(Action fileListUpdatedCallback)
-        {
-            FileListUpdateCallbacks.Add(fileListUpdatedCallback);
-        }
-
-        public void RemoveNotifyFileListUpdated(Action fileListUpdatedCallback)
-        {
-            FileListUpdateCallbacks.Remove(fileListUpdatedCallback);
+            EOSManager.Instance.GetPlayerDataStorageInterface().QueryFileList(ref options, null, OnQueryFileListCompleted);
         }
 
         //-------------------------------------------------------------------------
         /// <summary>(async) Begin file data download.</summary>
         /// <param name="fileName">Name of file.</param>
         /// <param name="downloadCompletedCallback">Function called when download is completed.</param>
-        public void StartFileDataDownload(string fileName, Action downloadCompletedCallback = null)
+        public void ReadFile(string fileName, Action downloadCompletedCallback = null)
```

Comment:
Instead of using `ReadFile`, to have parity with the other data manager, we should rename this `StartFileDataDownload` (or something akin to that).

Reply:
Fixed.